### DISCUSSION TITLE
Cannot make the defualt project a Local Vault?

### DISCUSSION THREAD

**Discussion by bjsvedin** [Kong/insomnia#6697] [Feedback]
What is the point of Local Projects/Vault when the default project, which is what I have stuck with for basically everything, can't be converted into a Local Vault?

**Comment by bjsvedin** [comment]
Another problem, I can't delete a remote collection at all. I have now 4 temp/test collections in the remote. They are useless, I don't need/want them, but I can't delete them.

**Comment by subnetmarco** [comment]
@bjsvedin There is a limitation on the default project in our data model, that we will remove so that that one default project can also be converted to local. This requires a migration on our end which is why it's not available yet.

You can export the whole project and re-import it in a local one in the meantime.

>  I can't delete a remote collection at all.

Works on my end. Are you the owner (or Admin) of that organization?

**Reply by bjsvedin** [reply]
Yes I am the owner of the organization. Where is it that I delete remote collections? I don't see anything in the web app, and from the desktop app all delete buttons only remove it from my local.

**Reply by bjsvedin** [reply]
NVM. I found the stupid delete button. I had never used sync before I was forced to send it up to the cloud and had no need to ever touch the icons on the bottom. Now that I have removed all my data from your servers, I am deleting the app and moving on to something else. Have a good day.

**Comment by ash-eng-spin** [comment]
Is there any movement on this? I'm on version 8.6.0. Using the Free plan to evaluate for my team. There is only 1 project available and I don't see any options to make this 1 project save to local vault vs instead of the cloud. 

Can you confirm that currently the Individual, Team and Enterprise plans have local vault for Projects, but Free does not?

Non-cloud storage is the only reason we are looking for alternatives to Postman.

**Reply by subnetmarco** [reply]
Even on a free plan you should be able to create more than one project, is that not the case?

**Reply by ash-eng-spin** [reply]
ah I just found how to create a new Project in Free and select local vault. I think I was misunderstanding the collaboration with only 1 project.

My bad, thanks for quick response.
